lms/djangoapps/instructor_task/config/waffle.py

```diff
@@ -0,0 +1,15 @@
+"""
+This module contains various configuration settings via
+waffle switches for the contentstore app.
+"""
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag, WaffleFlagNamespace
+
+# Waffle flag namespace for studio
+WAFFLE_STUDIO_FLAG_NAMESPACE = WaffleFlagNamespace(name=u'instructor_task')
```

Comment:
You want something like `instructor_task` instead of `studio` in the variable name.